scripts/qa_crowbarsetup.sh

```diff
@@ -3549,7 +3549,7 @@ function kill_pacemaker_remote
     local n=120
     # Check for nova-evacute log on all cluster nodes
     while [[ $n -gt 0 && $ret != 0 ]]; do
-        for((i=1; i<=$num_controllers; i++)) ; do
+        for i in $(seq 1 $num_controllers)  ; do
```

Comment:
NIT: extra space betweeen ''')''' and ''';'''

Reply:
thanks for review